addons/web_editor/static/src/js/wysiwyg/widgets/color_palette.js

```diff
@@ -71,7 +72,10 @@ const ColorPaletteWidget = Widget.extend({
         const $colorSection = this.$('.o_colorpicker_sections[data-color-tab="theme-colors"]');
         const $clpicker = $(colorpickerArch || `<colorpicker><div class="o_colorpicker_section" data-name="common"></div></colorpicker>`);
         $clpicker.find('button').addClass('o_we_color_btn');
-        $clpicker.appendTo($colorSection);
+        const $clpicker_theme = $(colorpickerThemeArch || `<colorpicker><div class="o_colorpicker_section o_colorpicker_section_top" data-name="theme"></div></colorpicker>`); 
+        $clpicker_theme.find('button').addClass('o_we_color_btn');
+        $clpicker_theme.appendTo($colorSection);
+        $clpicker.insertAfter($clpicker_theme);
```

Comment:
I'm confused as to why we need a whole new template and promise and stuff instead of just appending the custom-color-picker after the existing theme colors and adapting the css a little. Can you explain the reason?

Reply:
Separating the original full template with a new theme colors template would make it reusable independently from the other lines, if we later want to display it somewhere else or in another way

Reply:
We can move it into its own template when the use-case arises, let's not engineer around "We might need this later" unless we have a good reason to do so ;)